debian/rules.in

```diff
@@ -9,14 +9,36 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+CUDA_VER = 11.6
+CUDA_PREFIX= /usr/local/cuda-$(CUDA_VER)
+CUDA_OPT = $(shell echo ${DEB_BUILD_PROFILES} | grep -qw cuda \
+	&& echo --with-cuda=$(CUDA_PREFIX) || echo --without-cuda)
+
 %:
 	dh $@ 
 
 override_dh_auto_configure:
-	@top_top_srcdir@/contrib/configure-release-mt --prefix=/usr --enable-examples --with-java=no
+	./contrib/configure-release-mt --prefix=/usr \
+		--enable-examples --with-java=no \
+		$(CUDA_OPT)
 	chmod +x debian/rules
 
+# We may have a single package or two packages. We can't tell in advance.
+# Therefore set destdir for the default of multiple packages (that also
+# works for a single package):
+override_dh_auto_install:
+	dh_auto_install --destdir=debian/tmp
+
 override_dh_shlibdeps:
 	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
 
+saved_files = changelog control rules
 override_dh_auto_clean:
+	for f in $(saved_files); do \
+	  cp -a debian/$$f debian/$$f.sav; \
+	done
+	dh_auto_clean
+	for f in $(saved_files); do \
+	  if [ ! -r debian/$$f ]; then mv -v debian/$$f.sav debian/$$f; fi; \
+	  rm -f debian/$$f.sav; \
+	done
```

Comment:
why needed?

Reply:
That was a hack because of files getting deleted by the clean target.

Reply:
maybe update the clean target? or prevent it from being run?
is this a standard method for deb?

Reply:
The standard in Debian is that the clean target is run. And does not delete any essential deb packaging files.

Reply:
any idea why this happens? i couldn't find any debian clean steps in Makefile.am..